domain/src/main/java/com.pyshankov.microservices.domain/Product.java

```diff
@@ -1,13 +1,71 @@
 package com.pyshankov.microservices.domain;
 
+import org.springframework.data.annotation.Id;
+
 import java.math.BigDecimal;
 
-/**
- * Created by pyshankov on 4/29/18.
- */
 public class Product {
+    @Id
     private String id;
     private String name;
     private String description;
     private BigDecimal price;
+    private User owner;
```

Comment:
I think it will be better to store only userName there instead of the hole User object(userName is email)